src/OfficeHours.ts

```diff
@@ -61,15 +61,15 @@ export default class OfficeHours {
 
     public onAsk(message: Discord.Message, isAdmin: boolean, command: string, args: string[]) {
         const question = args.join(" ");
-        this.storeQuestion(question, message, message.author.id, message.author.toString());
+        this.storeQuestion(question, message, message.author.toString(), message.author.username);
     }
 
     public onAskFor(message: Discord.Message, isAdmin: boolean, command: string, args: string[]) {
         const asked = message.mentions.members.first();
         if (!asked) return;
```

Comment:
This makes sense in this boolean-esque situation, but is weird below (asked.id and asked.toString()). Can you change it to `asker`?